### ISSUE TITLE
Similar PIXOR implementation

### ISSUE THREAD

**Issue by philip-huang** [overfitover/pixor_pytorch#4]
Hi,

My name is Philip Huang and I have been working on reproducing PIXOR as a coding project for a machine learning course and the U of T self-driving team involved in the AutoDrive competition since last September.

Actually,  @ankita-kalra forked an uncompleted implementation from my repo durinig last October. So if you go to check commit history in her repo, you will find my name!) The old code did not include many of the most recent updates I had (including a bunch of bug fixes, tensorboardX, benchmarking results and improved mAP from around 15% to 60% on the validation set).

You can find some of my later edits here (https://github.com/philip-huang/PIXOR) . I am really looking forwards to fork/commit/edit to improve my result. At the end of the day, everyone (including me) would benefit from our collective work if we can train a good model 👍 )

Hope this help you guys as well

Philip

**Comment by ankita-kalra** [comment]
Hi all, 
The original implementation is by Philip-Huang. I stopped working on it after sometime. It would be great if any credits regarding that repository is given to Philip. 

Thanks

**Comment by overfitover** [comment]
I am sorry. I don't know about this situation. Thank you for your work. @philip-huang @ankita-kalra
